lib/ts/framework/custom/nodeHeaders.ts

```diff
@@ -64,7 +64,7 @@ export default class Headers extends URLSearchParams {
         } else if (init == null) {
             // eslint-disable-line no-eq-null, eqeqeq
             // No op
-        } else if (typeof init === "object" && !types.isBoxedPrimitive(init)) {
+        } else if (typeof init === "object" && isBoxedPrimitive(init)) {
```

Comment:
why did this change?

Reply:
`util` from where `types` was being imported is not supported on `edge`.

Reply:
right, but the condition is flipped.

Reply:
Ohh, my bad. Let me fix that 😅

Reply:
Done